### PR TITLE
[DNM] tikv-server: get cluster id from PD

### PR DESCRIPTION
**Pull request by huachaohuang**
Get the cluster ID from PD when create the `RpcClient`.

### CODE REVIEW

**Comment by huachaohuang** [comment]
PTAL @siddontang @BusyJay

File: src/pd/protocol.rs [hunk 1]
```diff
@@ -17,6 +17,16 @@ use protobuf::RepeatedField;
 use super::{Error, Result, RpcClient};
 
 impl super::PdClient for RpcClient {
+    fn get_cluster_id(&self) -> Result<u64> {
+        let get_pd_members = pdpb::GetPDMembersRequest::new();
```
Comment (siddontang) [inline src/pd/protocol.rs]: we should add some comments to explain why we can use GetPDMembers here.

**Comment by huachaohuang** [comment]
Restart CI after https://github.com/pingcap/pd/pull/370 has been merged.

File: src/bin/tikv-server.rs [hunk 2]
```diff
@@ -882,7 +880,6 @@ fn main() {
                 "dsn",
                 "[deprecated] set which dsn to use, warning: now only support raftkv",
                 "dsn: raftkv");
-    opts.optopt("I", "cluster-id", "set cluster id", "must greater than 0");
```
Comment (siddontang) [inline src/bin/tikv-server.rs]: how about old user pass --cluster-id here,  panic?

Comment (siddontang) [inline src/bin/tikv-server.rs]: I think we should keep the flag but add "deprecated" now.

**Comment by siddontang** [comment]
LGTM

**Comment by BusyJay** [comment]
LGTM, please fix ci.

**Pull request merged**